wsl-vpnkit-unsetup.sh

```diff
@@ -4,50 +4,23 @@ set -eu
 
```

Comment:
I would think we can keep in the rm_if lines as they are for backwards compatibility